docs/portal/registry-users/changing-passwords.md

```diff
@@ -0,0 +1,8 @@
+# Changing Passwords
+
+ECP allows you to change the passwords of registry users. A valid password must be 8 to 20 characters long and have at least one lowercase letter, one uppercase letter, and one number. To make sure you change the password of the appropriate user, you can view the user's information before changing the user's password. 
+1. In the left pane, click **Image Registry > Registry Users**.
+2. Perform one of the following tasks:<ul><li>To change the password from the Registry Users page, click its **Actions** button and select **Change Password**. </ul></li> <ul><li>To view information about a user before changing the user's password, click a username to display the Edit page. To change the password, click the **Change Password** button at the top right.</ul></li>
```

Comment:
The 2 tasks listed do not have same indention.
Please change “to display the Edit page” to “to display the detail page” for reason stated in comments above.

Reply:
Fixed.